client/src/components/NewSurveyForm.js

```diff
@@ -0,0 +1,39 @@
+import React, { useState, useCallback } from "react";
+
+import { Form, FormLayout, TextField, Button, Card } from "@shopify/polaris";
+
+export function NewPageForm() {
+  const [id, setId] = useState("");
+  const [title, setTitle] = useState("");
```

Comment:
where is the state coming from?

Reply:
React's `useState` manages this state in memory.

Reply:
I just don't get how your getting all these variables from the inital state which is empty string

Reply:
You can get see the react docs on [state](https://reactjs.org/docs/hooks-state.html#declaring-a-state-variable) for more details.  See the `What does useState return?` section